src/nft/hooks/useIsTablet.ts

```diff
@@ -0,0 +1,28 @@
+import { breakpoints } from 'nft/css/sprinkles.css'
+import { useEffect, useState } from 'react'
+
+const isClient = typeof window === 'object'
```

Comment:
Use `window !== undefined` instead of `window === 'object'`.

I'd consider it more idiomatic, because its documented by at least one other framework ([nextjs](https://nextjs.org/docs/migrating/from-create-react-app#safely-accessing-web-apis)).

Reply:
Made this change for both isTablet and isMobile